packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -724,8 +724,14 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
                         // This looks like a data corruption but the culprit has been found instead
                         // to be the file being overwritten in storage.  See PR #5882.
```

Comment:
Please reconcile with this comment - I added it some random time I was looking at this error :P  Your message is better so probably just delete it (maybe move it up here though)